clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -260,8 +267,8 @@ static String formatTableRequestPath(Namespace ns) {
    */
   static void checkTableNamespace(Namespace namespace) {
     Namespace.check(
-        namespace != null && namespace.length() == 3,
-        "Table namespace must be non-null and have 3 level, the input namespace is %s",
+        namespace != null && namespace.length() == 1,
```

Comment:
For metadata table, we may have more than 1 level namespace.

Reply:
How to support metadata table is still under discussion (lead by Xiaojing). So I just keep rule not changed here. Later when the plan is decided, we can change the logic in all places.